packages/lucide-svelte/package.json

```diff
@@ -71,6 +71,6 @@
     "vitest": "^1.1.1"
   },
   "peerDependencies": {
-    "svelte": ">=3 <5"
+    "svelte": "^3 || ^4 || ^5.0.0-next.0"
```

Comment:
Maybe we should specify which version it was fixed?

Reply:
Good catch!